src/aiod_sdk/calls/calls.py

```diff
@@ -0,0 +1,165 @@
+import aiohttp
+import asyncio
+import requests
+
+import pandas as pd
+from typing import Literal
+from functools import partial
+
+from aiod_sdk.calls.utils import (
+    url_to_get_asset,
+    url_to_get_list,
+    url_to_resource_counts,
+    format_response,
+    wrap_calls,
+)
+
+
+def get_list(
+    asset_type: str,
+    offset: int = 0,
+    limit: int = 10,
+    version: str | None = None,
+    data_format: Literal["pandas", "dict"] = "pandas",
+) -> pd.DataFrame | dict:
```

Comment:
```suggestion
def get_list(
    *,
    asset_type: str,
    offset: int = 0,
    limit: int = 10,
    version: str | None = None,
    data_format: Literal["pandas", "dict"] = "pandas",
) -> pd.DataFrame | dict:
```

We want to force people to use keywords here (and mention that in the docstring too). Otherwise, using positionals will actually raise an error:

```python
datasets = aiod.datasets.get_list(1, limit=2)
>>> TypeError: get_list() got multiple values for argument 'asset_type'
```
This happens because `asset_type` is already provided with the `partial`, but since we allow positionals, the first value is also assumed to be `asset_type`. This looks very strange to users, especially since the `asset_type` isn't even a parameter in the docstring!

Reply:
Same goes for all wrapped calls.

Reply:
hmmm that's strange, it doesn't raise a `TypeError` for me. But that happened before when `partial` was set with the keyword `asset_type`, which is no longer the case. In any case, I'm implementing your suggestion to the code.